generated/nidmm/nidmm.proto

```diff
@@ -557,7 +565,8 @@ message CheckAttributeViReal64Request {
   NiDmmAttribute attribute_id = 3;
   oneof attribute_value_enum {
     NiDmmReal64AttributeValues attribute_value = 4;
-    double attribute_value_raw = 5;
+    NiDmmReal64AttributeValuesMapped attribute_value_mapped = 5;
```

Comment:
Need to add field number overrides for this function to keep attribute_value_raw as 5